### ISSUE TITLE
Backtester - documentation request

### ISSUE THREAD

**Issue by knopki** [michaelgrosner/tribeca#59]
Readme and wiki page says that Tribeca packaged with backtesting mode. But how to run and use it?

**Comment by ctubio** [comment]
hope this helps: https://github.com/michaelgrosner/tribeca/search?utf8=%E2%9C%93&q=inBacktestMode

**Comment by ahalekelly** [comment]
I agree, a bit of documentation on backtesting would  be very helpful.

**Comment by ahalekelly** [comment]
@michaelgrosner What market data do you use for backtesting? What format does Tribeca require?

**Comment by ahalekelly** [comment]
Looking at [this line](https://github.com/michaelgrosner/tribeca/blob/master/src/service/main.ts#L120) and [this section](https://github.com/michaelgrosner/tribeca/blob/master/src/common/models.ts#L38), it seems to take a JSON array of Markets (timestamped order books) or MarketTrades (timestamped trade records). All the Bitcoin market data I've found only has price data, so it seems like recording realtime data from the exchanges is the best bet.

**Comment by ctubio** [comment]
try installing mocha and run test/*.ts files, like:
`mocha --compilers ts:ts-node/register test/*.ts`

you can see the backtests running at https://travis-ci.org/ctubio/tribeca/jobs/194689928
